ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -333,7 +333,7 @@ class AlertsView extends React.Component {
             frameworkOptions={frameworkOptions}
             issueTrackers={issueTrackers}
             optionCollectionMap={optionCollectionMap}
-            fetchAlertSummaries={(id, update, page) =>
+            fetchAlertSummaries={(id, update = true, page) =>
```

Comment:
This is a solution, but I think
```
fetchAlertSummaries={(id, page) =>
 this.fetchAlertSummaries(id, true, page)
```
Would restore the previous, good behaviour.

Reply:
I tested things locally and it seems that the best solution was the first you provided.
When changing pages `update` params is `false` and must be `false`.
But, when clicking on `Acknowledge`, `update` must be `true`, to fix the current issue.
So the solution would be:
```
fetchAlertSummaries={(id, update = true, page) =>
              this.fetchAlertSummaries(id, update, page)
            }
```

Reply:
I might be reading this wrong but wouldn't it be better to have `update=true` as the last param in the event someone wanted to pass `false`. Passing in that arg looks optional otherwise, and then omitting that second argument would cause a bug right? 
```
fetchAlertSummaries={(id, page, update = true) =>
              this.fetchAlertSummaries(id, update, page)
            }
```